src/Core/Services/MetadataProviders/MsSqlMetadataProvider.cs

```diff
@@ -79,5 +79,21 @@ public override async Task PopulateTriggerMetadataForTable(string entityName, st
                 }
             }
         }
+
+        /// <inheritdoc/>
+        public override DbType? ResolveDbType(string sqlDbTypeName)
```

Comment:
I think having ResolveDbType in SqlMetaDataProvider will allow us to have overridden implementations for getting the `DbType` from the pgsql/mysql/(probably dwsql?) data type if there is a need in future. This will help us avoid having to write specific hardcoded handlers in the TypeHelper class for each of the database flavors.

Reply:
For now, moved the method to MsSqlMetadataProvider.